### PR TITLE
nyx-ci: init

### PR DESCRIPTION
**Pull request by JustTNE**

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        with:
+          extra_nix_config: "accept-flake-config = true"
+          nix_path: "nixpkgs=channel:nixpkgs-unstable"
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Check nixpkgs-fmt
+        run: |
+          nix shell nixpkgs#nixpkgs-fmt -c nixpkgs-fmt --check ${{ github.workspace }}
```
Comment (PedroHLC) [inline .github/workflows/pull-request.yml:26]:
```suggestion
          nix run nixpkgs#nixpkgs-fmt -- --check ${{ github.workspace }}
```

File: .github/workflows/needs-cache.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        uses: cachix/cachix-action@v12
+        with:
+          name: chaotic-nyx
+          authToken: ${{ secrets.CACHIX_AUTH_TOKEN }}
+      - name: Upload to cachix
+        run: |
+          cd "${{ runner.temp }}"
+          cat push.txt | cachix push chaotic-nyx \
+            --compression-method zstd
+  organize:
```
Comment (PedroHLC) [inline .github/workflows/needs-cache.yml:38]: Can it push a comment with the failure list?

**Review by dr460nf1r3 — DISMISSED** [review]
Magnificence.

**Review by PedroHLC — APPROVED** [review]

**Pull request merged**